src/main/python/interfaces/widgets/lineedit.py

```diff
@@ -37,12 +42,24 @@ def __init__(
         echo is True and self.setEchoMode(QLineEdit.Password)
 
     def enable(self):
+        """
+        Enable input.
+        :return: this function doesn't return value
+        """
         self.setEnabled(True)
 
     def disable(self):
+        """
+        Disable input.
+        :return: this function doesn't return value
+        """
         self.setEnabled(False)
 
     def reset(self):
+        """
+        Clean up input text, remove focus, and enable input.
+        :return: this function doesn't return value
```

Comment:
reset can mean a lot. And you can see the `reset` at `label` and `lineedit` work differently. It might be confuse when they go together. For example:
```python
hint.reset()
submit.reset()
table.reset()
```

Reply:
see above - i just mean that `:return: this function doesn't return value` can be eliminated

Reply:
Fixed